dissect/target/plugins/os/windows/tasks.py

```diff
@@ -111,7 +111,6 @@ class TasksPlugin(Plugin):
         "unified_scheduling_engine",
         "disallow_start_on_remote_app_session",
         "data",
-        "_target",
     ]
```

Comment:
`TaskRecord.fields().keys()` should give you similar output, so you don't need to duplicate it here.